File: SisifoNotes.Lib/Models/Event.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SisifoNotes.Lib.Models
{
    public class Event : Note
    {
        public DateTime DateInit { get; set; }
        public DateTime DateFinish { get; set; }
        public string Place { get; set; }
        public string Description { get; set; }
        
    }
}

File: SisifoNotes.Web/wwwroot/Scripts/Models/Event.js
class Event extends Note
{
    constructor(json)
    {
        super(json)
        if (json)
        {
            this.DateInit = json.dateInit;
            this.DateFinish = json.dateFinish;
            this.Place = json.place;
            this.Description = json.description;
        }
        else
        {
            this.DateInit = Date();
            this.DateFinish = Date();
            this.Place = "";
            this.Description = "";
        }
    }
}